src/pages/home/home.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import "./home.css"
+import Header from "../../components/header/Header";
+import redditHead from "../../assets/logo.png"
+import Tile from "../../components/Tile/Tile";
+import useFetch from "../../customHooks/useFetch";
+
+function Home(props) {
+    const [data, setData] = useState(undefined);
```

Comment:
Als je de constructor leeg laat dan is het standaard al undefined
```suggestion
    const [data, setData] = useState();
```